modules/luponmediaBidAdapter.js

```diff
@@ -315,15 +315,6 @@ function newOrtbBidRequest(bidRequest, bidderRequest, currentImps) {
         format: bannerSizes
       }
     }]),
-    ext: {
```

Comment:
same as above, also looks like a bid adapter so I'm not sure why we'd need to remove it here